src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+
+    private Excavator dniproExcavator = new Excavator();
+    private Excavator kharkivExcavator = new Excavator();
+    private Excavator johnDeerExcavator = new Excavator();
+
+    private List<Excavator> excavatorList = List.of(dniproExcavator,
+             kharkivExcavator, johnDeerExcavator);
```

Comment:
`to produce` means to create new objects each time when we are trying to get them, so please move this logic to the `get` method
```suggestion
```